src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -314,6 +316,7 @@ public void Push(Page page)
 
                     //Invoke Page events
                     curTop.InvokeDisappeared();
+                    NotifyAccessibilityStatesChangeOfPages(curTop, null);
```

Comment:
I think that this line needs to be removed.
Instead, the following code is required at line 332.
```
NotifyAccessibilityStatesChangeOfPages(curTop, page);
```

Reply:
done